src/rpcblockchain.cpp

```diff
@@ -5419,6 +5426,8 @@ json_spirit::Value rpc_getblockstats(const json_spirit::Array& params, bool fHel
         result.push_back(Pair("first_time", TimestampToHRDate(l_first_time)));
         result.push_back(Pair("last_time", TimestampToHRDate(l_last_time)));
         result.push_back(Pair("time_span_hour", ((double)l_last_time-(double)l_first_time)/(double)3600));
+        result.push_back(Pair("min_blocksizek", size_min_blk/(double)1024));
```

Comment:
Please change to just a double literal since it needs to be done later anyway.

The field should be "min_blocksize", not " min_blocksizek", right?